plugins/module_utils/rds.py

```diff
@@ -195,13 +230,9 @@ def wait(client, db_instance_id, waiter_name, extra_retry_codes):
         'db_instance_stopped': 'stopped',
     }
     expected_status = waiter_expected_status.get(waiter_name, 'available')
-    if expected_status == 'available':
-        extra_retry_codes = ['DBInstanceNotFound']
```

Comment:
I don't see this added back anywhere, should it be added in rds_instance as part of https://github.com/ansible-collections/community.aws/pull/1081? (Of course not when the instance is being deleted)

Reply:
The `extra_retry_codes` here were never used, so that's why they're not added back :)